static/Gruntfile.js

```diff
@@ -12,34 +12,30 @@ module.exports = function (grunt) {
     watch: {
       all: {
         files: ['**', '!**/node_modules/**', `!${outputDir}/**`],
-        tasks: ['jambobuild', 'webpack',],
-        options: {
-          spawn: false,
-        },
+        tasks: ['build-site']
       },
     },
   });
 
   grunt.loadNpmTasks('grunt-webpack');
```

Comment:
we can remove grunt-webpack

Reply:
Tom and I figured that we should keep grunt webpack because the ci scripts call `grunt webpack`